src/frontend/src/footer/style.css

```diff
@@ -0,0 +1,89 @@
+footer {
+  background-color: var(--semi-dark);
+  text-align: left;
+  font-size: 14px;
+}
+footer.open {
+  max-height: 500px; 
+}
+footer .footer-container{
+  display: flex;
+  justify-content: space-between;
+  max-width: 900px;
+  margin: 0 auto;
+}
```

Comment:
please put styles which are in footer-container inside {}. 
also apply this code style to all places. 
example: 
if you have 
<div class="footer-container">
  <div class="logo"> 
  </div>
</div>

it should look like 
.footer-container{
  display: flex;
   
  .logo{
    justify-content: ...
  }
}

Reply:
and try to do in all places where it is needed

Reply:
in this case you won't overwrite other divs with the same names

Reply:
and please, resolve conflicts